js/src/Ice/ObjectAdapter.js

```diff
@@ -199,9 +199,6 @@ export class ObjectAdapter {
     }
 
     getName() {
-        //
-        // No mutex lock necessary, _name is immutable.
```

Comment:
While true, this comment is lame (and probably just a copy/paste from the other languages).
Of course we don't need a mutex in JS.